modules/tinymce/src/core/test/ts/browser/content/EditorContentTest.ts

```diff
@@ -27,75 +28,191 @@ UnitTest.asynctest('browser.tinymce.core.content.EditorContentTest', (success, f
+
+      Log.stepsAsStep('TINY-6281', 'getContent html with empty editor', [
+        sClearEvents(),
+        tinyApis.sSetContent(''),
+        Step.sync(() => {
+          const html = editor.getContent();
+          Assertions.assertHtml('Should be expected html', '', html);
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TINY-6281', 'getContent text', [
+        sClearEvents(),
+        tinyApis.sSetContent('<p>Text to be retrieved</p>'),
+        Step.sync(() => {
+          const text = <string> editor.getContent({ format: 'text' });
+          Assertions.assertHtml('Should be expected text', 'Text to be retrieved', text);
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TINY-6281', 'getContent text with empty editor', [
+        sClearEvents(),
+        tinyApis.sSetContent(''),
+        Step.sync(() => {
+          const text = editor.getContent({ format: 'text' });
+          Assertions.assertHtml('Should be expected text', '', text);
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TBA', 'getContent tree', [
+        sClearEvents(),
         tinyApis.sSetContent('<p>tree</p>'),
         Step.sync(() => {
-          const tree = EditorContent.getContent(editor, { format: 'tree' }) as AstNode;
+          const tree = editor.getContent({ format: 'tree' }) as AstNode;
           Assertions.assertHtml('Should be expected tree html', '<p>tree</p>', toHtml(tree));
-        })
-      ])),
-      Logger.t('getContent tree filtered', GeneralSteps.sequence([
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TINY-6281', 'getContent tree with empty editor', [
+        sClearEvents(),
+        tinyApis.sSetContent(''),
         Step.sync(() => {
-          EditorContent.setContent(editor, '<p><font size="7">x</font></p>', { format: 'raw' });
-          const tree = EditorContent.getContent(editor, { format: 'tree' }) as AstNode;
+          const tree = editor.getContent({ format: 'tree' }) as AstNode;
+          // bogus br that sits in an empty editor is replaced with a &nbsp; by the html parser, hence the space
+          Assertions.assertHtml('Should be expected tree html', '<p> </p>', toHtml(tree));
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TBA', 'getContent tree filtered', [
+        sClearEvents(),
+        Step.sync(() => {
+          editor.setContent('<p><font size="7">x</font></p>', { format: 'raw' });
+          const tree = editor.getContent({ format: 'tree' }) as AstNode;
           Assertions.assertHtml('Should be expected tree filtered html', '<p><span style="font-size: 300%;">x</span></p>', toHtml(tree));
-        })
-      ])),
-      Logger.t('getContent tree using public api', GeneralSteps.sequence([
+        }),
+        sAssertEventsFiredInOrder()
+      ]),
+
+      Log.stepsAsStep('TBA', 'getContent tree using public api', [
```

Comment:
Is this step still needed if the above `'getContent tree'` step is doing the same thing since it was changed to use `editor.getContent`?